frontend/src/components/contact/ContactHistoryList.vue

```diff
@@ -58,9 +63,9 @@ function getRouteToObject(data: Enquiry | HousingProject) {
 function normalizeContactHistory() {
   return contactsHistory.map((se) => ({
     ...se,
-    state: 'housingProjectId' in se ? se.applicationStatus : se.enquiryStatus,
+    state: 'projectId' in se ? se.applicationStatus : se.enquiryStatus,
     assignedUser: se.assignedUserId ? getUsersName(se.assignedUserId) : 'Unassigned',
-    historyType: 'housingProjectId' in se ? 'Project' : 'Enquiry'
+    historyType: 'projectId' in se ? 'Project' : 'Enquiry'
```

Comment:
Could "future proof" and have the true case return '<Initiative> Project', even if just a small chance it of props being in both... But even then, it may be useful for Navs to see the Initiative that is in the prop's history without clicking into it.